packages/medusa-plugin-sanity/src/loaders/check-types.js

```diff
@@ -0,0 +1,76 @@
+const checkContentTypes = async (container) => {
+  const contentfulService = container.resolve("contentfulService")
+
+  let product
+  let variant
+
+  try {
+    product = await contentfulService.getType("product")
+    variant = await contentfulService.getType("productVariant")
+  } catch (error) {
```

Comment:
`contentfulService` is not available here

Reply:
https://vuuzletvph.com/what-is-the-secret-of-nft-tokens-and-why-they-are-bought-for-millions-of-dollars/

Kindly check this out.Have a great day ahead!

Reply:
https://vuuzletvph.com/heads-up-vuco-bounty-programs-are-coming-your-way/